src/current/_redirects

```diff
@@ -560,6 +560,7 @@ layout: null
 /docs/v21.1/build-a-php-app-with-cockroachdb.html /docs/stable/community-tooling.html 301
 /docs/stable/hello-world-example-apps.html /docs/stable/example-apps.html 301
 /docs/cockroachcloud/hello-world-example-apps.html /docs/stable/example-apps.html 301
+/docs/cockroachcloud/cmek-faq.html /docs/cockroachcloud/cmek.html#faqs 301
```

Comment:
```suggestion
/docs/cockroachcloud/cmek-faq.html /docs/cockroachcloud/cmek#faqs 301
/docs/cockroachcloud/cmek-faq /docs/cockroachcloud/cmek#faqs 301
```

Changed my mind. Found a bug where Marketing was using URLs without extensions, so we need to start accounting for that.